Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py

```diff
@@ -0,0 +1,2138 @@
+            'Type': owner['type'],
+            'Name': owner['name']
+        })
+
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['json'],
+        'Contents': raw_owners,
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': tableToMarkdown('ThreatConnect Owners:', owners),
+        'EntryContext': {'TC.Owner(val.ID && val.ID === obj.ID)': owners}
+    })
+
+
+def tc_owners():
+    tc = get_client()
+    ro = RequestObject()
+    ro.set_http_method('GET')
+    ro.set_request_uri('/v2/owners')
+    results = tc.api_request(ro)
+
+    return results.json()
+
+
+def tc_get_indicator_owners():
+    owners = []
+    ownersRaw = get_indicatorOwner(demisto.args()['indicator'])
+    if 'status' in ownersRaw:
+        if ownersRaw['status'] == 'Success':
+            if len(ownersRaw['data']['owner']) > 0:
+                owners = ownersRaw['data']['owner']
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['json'],
+        'Contents': ownersRaw,
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': tableToMarkdown('ThreatConnect Owners for Indicator:' + demisto.args()['indicator'], owners),
+        'EntryContext': {'TC.Owners': owners}
+    })
+
+
+def tc_associated_groups(tc, owners, indicator_value, indicator_type):
+    group_associations = []
+    types = tc_get_indicator_types_request()['data']['indicatorType']
+    for item in types:
+        if indicator_type is not None:
+            if item['name'] == indicator_type:
+                apiBranch = item['apiBranch']
+        else:
+            # meaning we got an indicator but SDK returned a null type
+            apiBranch = None
+
+    ro = RequestObject()
+    ro.set_http_method('GET')
+    ro.set_owner(owners)
+    if apiBranch is not None:
+        ro.set_request_uri("/v2/indicators/{}/{}/groups".format(apiBranch, indicator_value))
+        results = tc.api_request(ro)
+        if results.headers['content-type'] == 'application/json':
+            if 'data' in results.json():
+                if 'group' in results.json()['data']:
+                    group_associations = results.json()['data']['group']
+            else:
+                group_associations = []
+    else:
+        try:
+            for item in types:
+                ro.set_request_uri(
+                    "/v2/indicators/{}/{}/groups".format(item['apiBranch'], quote(indicator_value, safe='')))
+                results = tc.api_request(ro)
+                if results.headers['content-type'] == 'application/json':
+                    if 'data' in results.json():
+                        if 'group' in results.json()['data']:
+                            group_associations = results.json()['data']['group']
+                            break
+                    else:
+                        group_associations = []
+        except Exception:
+            pass
```

Comment:
never ignore an exception completely, at the very least write it to the log (`demisto.error`)